src/mbgl/layout/pattern_layout.hpp

```diff
@@ -18,15 +18,20 @@ using PatternLayerMap = std::map<std::string, PatternDependency>;
 
 class PatternFeature  {
 public:
-    const uint32_t i;
+    friend bool operator < (const PatternFeature& lhs, const PatternFeature& rhs) {
```

Comment:
nit: const method

Reply:
"Non-member function cannot have 'const' qualifier"
Which I think is valid, and lhs and rhs are both const.